### ISSUE TITLE
build issue

### ISSUE THREAD

**Issue by yuanshuaipeng123** [hzeller/gmrender-resurrect#212]
Hi, sorry for disturbing you,
could you pls share me the code build command?

Cause I am a build newer, some issue occurred such as :

Useless use of /d modifier in transliteration operator at /usr/local/share/automake-1.9/Automake/Wrap.pm line 60.
configure.ac:3: required directory ./config does not exist
configure.ac: installing `config/install-sh'; error while making link: No such file or directory
configure.ac: installing `config/missing'; error while making link: No such file or directory
src/Makefile.am: installing `config/depcomp'; error while making link: No such file or directory

thanks for responding.

**Comment by hzeller** [comment]
You didn't describe what commands you invoked when you saw this error.

Did you follow 
https://github.com/hzeller/gmrender-resurrect/blob/master/INSTALL.md
?

**Comment by yuanshuaipeng123** [comment]
does the proj can run on arm linux?thx.

**Comment by hzeller** [comment]
Yes, definitely. People run it regularly on a Raspberry Pi and other small ARM-based systems.

**Comment by JasonLG1979** [comment]
On a Raspberry Pi Zero running an up to date version of Raspberry Pi OS Lite as of the date of this comment.

As of https://github.com/hzeller/gmrender-resurrect/commit/26d8f7edf5336bc30f7334c6ad459bf7f4f90ff2 the instructions in https://github.com/hzeller/gmrender-resurrect/blob/master/INSTALL.md are missing two dependencies. ``libglib2.0-dev libgstreamer-plugins-base1.0-dev `` and ``./configure`` needs to be ran with `` --with-gstreamer`` otherwise you get ``error "No output configured. You need to ./configure --with-gstreamer"``

After installing the missing dependencies and using the ```--with-gstreamer``` everything builds, installs and works fine.

**Comment by JasonLG1979** [comment]
Currently streaming Spotify from an old 5th gen Amazon Fire 7 tablet running LineageOS 12.1 with Xposed audiocast and BubbleUPNP 3.4.10.1 to my Raspberry Pi Zero W to a USB DAC to my stereo. Sounds a heck of a lot better than Bluetooth on the same hardware.

**Comment by mill1000** [comment]
Can confirm that on a fresh Raspberry Pi OS Lite the additional dependencies were required. 

Once installed, the configure script correctly detected GST and it was not necessary to specify `--with-gstreamer`.

**Comment by parasange** [comment]
I've installed all dependencies mentioned in INSTALL.md for Raspbian Buster on Raspberry Zero, and used `./configure --with-gstreamer` but `make` still throws the error 

> output.c:51:2: error: #error "No output configured. You need to ./configure --with-gstreamer"
 #error "No output configured. You need to ./configure --with-gstreamer"


Output of `./autogen.sh`:

```
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory config
autoreconf: configure.ac: not using Libtool
autoreconf: running: /usr/bin/autoconf --force
autoreconf: running: /usr/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:8: installing 'config/compile'
configure.ac:6: installing 'config/install-sh'
configure.ac:6: installing 'config/missing'
Makefile.am: installing './INSTALL'
src/Makefile.am: installing 'config/depcomp'
autoreconf: Leaving directory `.'
```

```
Output of `./configure --with-gstreamer`:
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... no
checking for mawk... mawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... 64
checking for gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking dependency style of gcc... (cached) gcc3
checking for gcc option to accept ISO C99... none needed
checking for gcc option to accept ISO Standard C... (cached) none needed
checking whether make sets $(MAKE)... (cached) yes
checking for asprintf... yes
checking for exp in -lm... yes
checking for pkg-config... /usr/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for GLIB... yes
checking for GST... no
checking for GST... no
checking for LIBUPNP... yes
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for ANSI C header files... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating data/Makefile
config.status: creating config.h
config.status: executing depfiles commands
```

**Comment by parasange** [comment]
> I've installed all dependencies mentioned in INSTALL.md for Raspbian Buster on Raspberry Zero, and used `./configure --with-gstreamer` but `make` still throws the error
> 
> > output.c:51:2: error: #error "No output configured. You need to ./configure --with-gstreamer"
> > #error "No output configured. You need to ./configure --with-gstreamer"
> 
> Output of `./autogen.sh`:
> 
> ```
> autoreconf: Entering directory `.'
> autoreconf: configure.ac: not using Gettext
> autoreconf: running: aclocal --force 
> autoreconf: configure.ac: tracing
> autoreconf: configure.ac: creating directory config
> autoreconf: configure.ac: not using Libtool
> autoreconf: running: /usr/bin/autoconf --force
> autoreconf: running: /usr/bin/autoheader --force
> autoreconf: running: automake --add-missing --copy --force-missing
> configure.ac:8: installing 'config/compile'
> configure.ac:6: installing 'config/install-sh'
> configure.ac:6: installing 'config/missing'
> Makefile.am: installing './INSTALL'
> src/Makefile.am: installing 'config/depcomp'
> autoreconf: Leaving directory `.'
> ```
> 
> Output of `./configure --with-gstreamer`:
> 
> ```
> checking for a BSD-compatible install... /usr/bin/install -c
> checking whether build environment is sane... yes
> checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
> checking for gawk... no
> checking for mawk... mawk
> checking whether make sets $(MAKE)... yes
> checking whether make supports nested variables... yes
> checking whether make supports the include directive... yes (GNU style)
> checking for gcc... gcc
> checking whether the C compiler works... yes
> checking for C compiler default output file name... a.out
> checking for suffix of executables... 
> checking whether we are cross compiling... no
> checking for suffix of object files... o
> checking whether we are using the GNU C compiler... yes
> checking whether gcc accepts -g... yes
> checking for gcc option to accept ISO C89... none needed
> checking whether gcc understands -c and -o together... yes
> checking dependency style of gcc... gcc3
> checking for special C compiler options needed for large files... no
> checking for _FILE_OFFSET_BITS value needed for large files... 64
> checking for gcc... (cached) gcc
> checking whether we are using the GNU C compiler... (cached) yes
> checking whether gcc accepts -g... (cached) yes
> checking for gcc option to accept ISO C89... (cached) none needed
> checking whether gcc understands -c and -o together... (cached) yes
> checking dependency style of gcc... (cached) gcc3
> checking for gcc option to accept ISO C99... none needed
> checking for gcc option to accept ISO Standard C... (cached) none needed
> checking whether make sets $(MAKE)... (cached) yes
> checking for asprintf... yes
> checking for exp in -lm... yes
> checking for pkg-config... /usr/bin/pkg-config
> checking pkg-config is at least version 0.9.0... yes
> checking for GLIB... yes
> checking for GST... no
> checking for GST... no
> checking for LIBUPNP... yes
> checking how to run the C preprocessor... gcc -E
> checking for grep that handles long lines and -e... /usr/bin/grep
> checking for egrep... /usr/bin/grep -E
> checking for ANSI C header files... yes
> checking that generated files are newer than configure... done
> configure: creating ./config.status
> config.status: creating Makefile
> config.status: creating src/Makefile
> config.status: creating data/Makefile
> config.status: creating config.h
> config.status: executing depfiles commands
> ```

Found the error. Accidentally installed libgstreamer1.0-plugins-base instead of gstreamer1.0-plugins-base
